addons/account/demo/account_demo.py

```diff
@@ -316,6 +323,19 @@ def _get_demo_data_statement(self, company=False):
                     }),
                 ]
             },
+            'demo_credit_statement_1': {
+                'name': f'{ccd_journal.name} - {time.strftime("%Y")}-01-01/1',
+                'balance_end_real': -1055.0,
+                'balance_start': 0.0,
+                'line_ids': [
+                    Command.create({
+                        'journal_id': ccd_journal.id,
+                        'payment_ref': 'Initial balance',
+                        'amount': -1055.0,
```

Comment:
Would be nice to think of something that would work nicely with a bill and  bank statement, but PO's can see that later...